src/table/tableview/table-view.component.js

```diff
@@ -67,6 +68,7 @@ angular.module('patternfly.table').component('pfTableView', {
       // Need to deep watch changes in dtOptions and items
```

Comment:
onToggleColvis should be in the config object to be consistent with other callback functions.